src/index.ts

```diff
@@ -29,18 +39,30 @@ client.initializeApp({
   appId: config.clientAppID,
 });
 
-const app = express();
-const port = process.env.PORT || 3001;
+// cxn created via configs from ormconfig.ts
+createConnection()
+  .then(_ => {
+    // magic happens here
+    // <-------------->
+    routingUseContainer(Container);
+    const app = createExpressServer({
+      cors: true,
+      controllers: [EventController],
+    });
+    // <-------------->
 
-app.use(cors());
-app.use(express.json());
-app.use(express.urlencoded({ extended: true }));
-app.use(limiter);
+    const port = process.env.PORT || 3001;
 
-app.use('/api/user', UserRouter);
-app.use('/docs', DocsRouter);
-app.use('/api/auth', AuthRouter);
+    app.use(express.json());
+    app.use(express.urlencoded({ extended: true }));
+    app.use(limiter);
+    app.use(ErrorHandler);
```

Comment:
Error handler middleware should be put at the end of all the app.use() calls, if you do plan to keep it there. I think you want to remove it though?

Reply:
ah hm - that makes sense. not sure what I want to do yet